measure_manager.rb

```diff
@@ -136,7 +136,7 @@ def get_measure(measure_dir, force_reload)
       @measure_info[measure_dir] = {}
     end
     
-    if result
+    if result && !@measures[measure_dir].nil?
```

Comment:
@nllong this is a bug but should probably just set result to false on line 129, was this the blow up with no measure.rb file?